src/App.tsx

```diff
@@ -1,44 +1,80 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import 'bulma/bulma.sass';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
-
 import classNames from 'classnames';
+
 import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
 import { Loader } from './components/Loader';
+import { getUsers } from './api/users';
+
+import { User } from './types/User';
+import { Post } from './types/Post';
 
 export const App: React.FC = () => {
+  const [users, setUsers] = useState<User[]>([]);
+  const [selectedUser, setSelectedUser] = useState<User | null>(null);
+  const [selectedPost, setSelectedPost] = useState<Post | null>(null);
+  const [isReceiving, setIsReceiving] = useState<boolean>(true);
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(true);
```

Reply:
Better and more understandable naming.
You don't need set generic in useState for primitive values.